crates/ezpdb/src/lib.rs

```diff
@@ -22,7 +22,7 @@ pub fn parse_pdb<P: AsRef<Path>>(
     base_address: Option<usize>,
 ) -> Result<ParsedPdb, crate::error::Error> {
     let file = File::open(path.as_ref())?;
-    debug!("opening PDB");
+    //debug!("opening PDB");
```

Comment:
I kind of did a side channel here to figure out why this was needed. The debug logic is necessary for debugging my own issues in another tool. If you don't want it, you should configure the logger you're using to filter out this module/crate: https://docs.rs/env_logger/latest/env_logger/struct.Builder.html#method.filter_module

Reply:
Reverted the changes here, will use the env_logger fix.